lib/launcher.js

```diff
@@ -4,120 +4,78 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-const fs = require('fs');
-const os = require('os');
 const path = require('path');
 const carlo = require('carlo');
 const { rpc, rpc_process } = require('carlo/rpc');
 
-const removeFolder = require('rimraf');
-const util = require('util');
-
 const { Backend } = require('./backend.js');
 
-const fsCopyFile = fs.copyFile ? util.promisify(fs.copyFile) : function(source, target) {
-  require('fs-copy-file-sync')(source, target);
-  return Promise.resolve();
-};
-const fsMkdir = util.promisify(fs.mkdir);
-const fsMkdtemp = util.promisify(fs.mkdtemp);
-const fsWriteFile = util.promisify(fs.writeFile);
-
 process.on('unhandledRejection', error => {
   if (error.message.includes('Protocol error') && error.message.includes('Target closed'))
     process.exit(1);
   console.log('unhandledRejection', error.message);
 });
 
-async function launch(options) {
-  const userDataDir = await setupUserDataDir(options.configDir, options.doNotCopyPreferences);
-  const app = await carlo.launch({
-    bgcolor: '#242424',
-    userDataDir,
-    channel: ['stable', 'canary', 'chromium']
-  });
-  let pkg = null;
+async function launch() {
+  let app;
   try {
-    pkg = require(path.join(options.cwd, 'package.json'));
+    app = await carlo.launch({
+      bgcolor: '#242424',
+      channel: ['stable', 'canary', 'chromium'],
+      title: JSON.stringify({
+        cwd: process.cwd(),
+        argv: process.argv
+      })
+    });
   } catch (e) {
+    console.log(e.message);
```

Comment:
Only print it in case message is unexpected (Blha blah already running)

Reply:
done